storage/mounts.go

```diff
@@ -92,9 +93,14 @@ func commonMountResource(tpl Mount, s map[string]*schema.Schema) *schema.Resourc
 
 // NewMountPoint returns new mount point config
 func NewMountPoint(client *common.DatabricksClient, name, clusterID string) MountPoint {
+	// TODO: the executor is mutable probably needs to be refactored
+	executor := client.CommandExecutor()
```

Comment:
why can't this change be done within DatabricksClient#CommandExecutor method? it's more logical to have it there, mounts should not know anything about existence of CommandsAPI, it should only know about CommandExecutor interface.

Reply:
it creates a cyclical import:

import cycle not allowed
package github.com/databrickslabs/databricks-terraform/compute/acceptance (test)
	imports github.com/databrickslabs/databricks-terraform/common
	imports github.com/databrickslabs/databricks-terraform/compute
	imports github.com/databrickslabs/databricks-terraform/common